dot/state/storage_test.go

```diff
@@ -31,23 +30,28 @@ func newTestStorageState(t *testing.T) *StorageState {
 }
 
 func TestStorage_StoreAndLoadTrie(t *testing.T) {
-	storage := newTestStorageState(t)
-	ts, err := storage.TrieState(&trie.EmptyHash)
-	require.NoError(t, err)
+	t.Skip()
+	// FIX ME
```

Comment:
Will this be addressed in this PR? If not should probably link an issue representing the needed

Reply:
Pushed in my last commit :D